app/models/orchestration_template.rb

```diff
@@ -153,7 +153,7 @@ def md5=(_md5)
   end
 
   def self.calc_md5(text)
-    Digest::MD5.hexdigest(text) if text
+    Rails.application.config.digest_class.hexdigest(text) if text
```

Comment:
question:  Wouldn't the migration be site specific and therefore changing the algorithm would be a decision for the team maintaining the app, not necessarily ManageIQ GitHub?